src/docs/clients/react-native/manual-setup.mdx

```diff
@@ -216,14 +216,26 @@ import io.sentry.RNSentryPackage;
 
 public class MainApplication extends Application implements ReactApplication {
 
-    @Override
-    protected List<ReactPackage> getPackages() {
-        return Arrays.<ReactPackage>asList(
-                new MainReactPackage(),
-                new RNSentryPackage()
-        );
-    }
+  @Override
+  protected List<ReactPackage> getPackages() {
+    return Arrays.<ReactPackage>asList(
+        new MainReactPackage(),
+        new RNSentryPackage()
+    );
+  }
+}
+```
+
+```kotlin
+import io.sentry.RNSentryPackage
```

Comment:
package is `io.sentry.react`, Java code snippet is also wrong

not totally related but https://github.com/getsentry/sentry-react-native/issues/1333

Reply:
done